Mesh_3/doc/Mesh_3/Mesh_3.txt

```diff
@@ -1013,7 +1013,20 @@ Right: the mesh with added 1D-features along sharp edges and edges shared
 by at least two of the input polyhedral surfaces forming the complex.
 \cgalFigureCaptionEnd
 
+\subsubsection Mesh_3EdgesSizingField Sizing field for feature edges
 
+The following example shows how to generate a mesh from a polyhedral complex
+or a polyhedral surface with polyline features.
+Polyline features are covered by a set of so-called "protecting balls" which
+sizes are driven by the size component of `Mesh_edge_criteria_3`.
+The ideal size of the protecting balls can be computed using
+`CGAL::Sizing_field_with_aabb_tree` that helps start the feature protection
```

Comment:
Here we qualify with the namespace,  three lines above we don't  and just having a look in the User Manual I notice that we do not do it in a coherent way. I think it should be dropped everywhere. Unless it is `CGAL::Mesh_3::....` @sloriot ??